src/SFA.DAS.Assessor.Functions.ExternalApis/Assessor/AssessorServiceApiClient.cs

```diff
@@ -25,36 +24,27 @@ public class AssessorServiceApiClient : ApiClientBase, IAssessorServiceApiClient
             Client.BaseAddress = new Uri(options?.Value.ApiBaseAddress);
         }
 
-        public async Task UpdateStandardSummary()
+        public Task UpdateStandardSummary()
```

Comment:
Why have you removed all the async call calls will that even work?

Reply:
Yes you don't need it if there's just one.  Just return the task.

Reply:
https://medium.com/@deep_blue_day/long-story-short-async-await-best-practices-in-net-1f39d7d84050

This is an interesting article it describes what you are saying about performance due to code bloat, however it also recommends that if returning from within a using block you should always return await to prevent race conditions on disposable.

Reply:
Yes I had thought about the using statement but it is not actually required.  The dispose on the HttpRequestMessage is not required for the HttpContent type if it is not a stream which in our cases it's not.

It would be better to remove the using statement completely.  

I will add the async awaits back in if you find it confusing because after all code is written for humans but it is not necessary as isn't the using statements.